starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,68 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <div class="topclass">
+    
+    <h1 id="appleid">Apple Pie</h1>
+  </div>
+<div class="grandclass">
+  <br>
+  <br>
+  <p>This was my grandmother's apple pie recipe. I have never seen another one<br>
+  quite like it. It will always be my favorite and has won me several first place<br>
+  prizes in local competitions. I hope it becomes one of your favorites as well!</p>
+</div>
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+<div class="midphoto">
+  <br>
+<img src="images/recipe-info.png">
+</div>
+    <br>
+    <br>
+<hr>
 
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
+<br>
+<br>
+<br>
+<h3 >Ingredients</h3>
+<div class="ingreclass">
+ 
+  <p>-1 recipe pastry for a 9 inch double crust pie</p>
```

Comment:
This is the perfect example to use lists ( in this case we can use `ul` and replace bulletins with `-` ).